app/config/drupal-demo/install-dashboard.php

```diff
@@ -7,7 +7,14 @@
   throw new RuntimeException('Missing environment variable: INSTALL_DASHBOARD_USERS');
 }
 $users = explode(';', getenv('INSTALL_DASHBOARD_USERS'));
-CRM_Core_BAO_CMSUser::synchronize(FALSE);
+
+// WISHLIST: CMSUser::synchronize probably merits an API but lacks test coverage.
+if (is_callable(array('CRM_Core_BAO_CMSUser', 'synchronize'))) {
+  CRM_Core_BAO_CMSUser::synchronize(FALSE); // 4.6 and earlier
+}
+else {
+  CRM_Core_Config::singleton()->userSystem->synchronizeUsers(); // v4.7+
```

Comment:
As of https://github.com/civicrm/civicrm-core/commit/091db9085e779dc6b50f1644c721846437b6c844 the more concise way to write this would be `CRM_Utils_System::synchronizeUsers()`